file.txt

```diff
@@ -1,3 +1,4 @@
 Some text ;
 Another line
 New line
+4th line
```

Comment:
Hey, it seems better to keep words instead of numbers, what do you think?

Reply:
Oh nice idea, I did that 😄